identity.go

```diff
@@ -543,13 +596,22 @@ func loadCABundle(caAddr string) (*x509.CertPool, error) {
 
 		case StorageFile, "":
 			if bundle, err = os.ReadFile(caUrl.Path); err != nil {
-				return nil, err
+				return nil, nil, err
 			}
 
 		default:
-			return nil, fmt.Errorf("NO valid Cert location specified")
+			return nil, nil, errors.Errorf("invalid cert location, unsupported scheme: '%v'", caAddr)
 		}
+
 		pool.AppendCertsFromPEM(bundle)
-		return pool, nil
+
+		certs, err := certtools.LoadCert(bundle)
```

Comment:
Seems unnecessary to parse the certs twice. `x509.CertPool` is an insert-only structure. If we need a `CertPool` that is readable, we should make one and then allow the custom CertPool type to generate `x509.CertPools` as needed to satisfy other interfaces. Opportunistic change, not necessary now.